Election/Election/main.m

```diff
@@ -206,6 +205,26 @@ - (BOOL)pollsOpen {
 int main(int argc, const char * argv[]) {
     @autoreleasepool {
         
+        Election *AnimalPresidentsElection = [[Election alloc] init];
+        [AnimalPresidentsElection setElectionName:@"Animal President Election"];
+        
+        Contender *ApeLincoln = [[Contender alloc] init];
+        Contender *PandadrewJackson = [[Contender alloc] init];
+        Contender *BearackObama = [[Contender alloc] init];
+        
+        [ApeLincoln initWithName:@"Ape Lincoln"];
+        [PandadrewJackson initWithName:@"Pandadrew Jackson"];
+        [BearackObama initWithName:@"Bearack Obama"];
```

Comment:
Calling `initWithName` after calling init is going to reinitialize the object. You should do one or the other:

```objective-c
 Contender *ApeLincoln = [[Contender alloc] initWithName:@"Ape Lincoln"];
```